### PR TITLE
Move the repo use the latest analyzer packages

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

File: src/Microsoft.CodeQuality.Analyzers/CSharp/ApiDesignGuidelines/CSharpStaticHolderTypes.Fixer.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public class CSharpStaticHolderTypesFixer : CodeFixProvider
             }
         }
 
-        private async Task<Document> MakeClassStatic(Document document, ClassDeclarationSyntax classDeclaration, CancellationToken ct)
+        private static async Task<Document> MakeClassStatic(Document document, ClassDeclarationSyntax classDeclaration, CancellationToken ct)
```
Comment (jinujoseph) [inline src/Microsoft.CodeQuality.Analyzers/CSharp/ApiDesignGuidelines/CSharpStaticHolderTypes.Fixer.cs]: which rule is triggering this static changes ?

Comment (jinujoseph) [reply src/Microsoft.CodeQuality.Analyzers/CSharp/ApiDesignGuidelines/CSharpStaticHolderTypes.Fixer.cs]: Is it CA1822 ?

Comment (mavasani) [reply src/Microsoft.CodeQuality.Analyzers/CSharp/ApiDesignGuidelines/CSharpStaticHolderTypes.Fixer.cs]: Yes.

**Comment by mavasani** [comment]
Ping.

**Pull request merged**